src/Umbraco.Web/PublishedCache/NuCache/ContentStore.cs

```diff
@@ -235,11 +235,27 @@ public void ReleaseLocalDb()
             var lockInfo = new WriteLockInfo();
             try
             {
-                Lock(lockInfo);
-
-                if (_localDb == null) return;
-                _localDb.Dispose();
-                _localDb = null;
+                try{
+                    // Trying to lock could throw exceptions so always make sure to clean up.
+                    Lock(lockInfo);
+                }
+                catch { throw; }
```

Comment:
Again, no need to do a try/catch/throw in a try/finally, it does not swallow anything in a try/finally